library/src/main/java/com/novoda/downloadmanager/DownloadMigrator.java

```diff
@@ -4,6 +4,6 @@
 
 public interface DownloadMigrator {
 
-    void startMigration(File databaseFile);
+    void startMigration(String jobIdentifier, File databaseFile);
```

Comment:
Supplying a job identifier means that notifications for a given job share the same id, allowing the `NotificationDispatcher` to switch between states correctly.